scheduler/tests/test_data.py

```diff
@@ -193,7 +193,7 @@ class GenerateRidesTestData:
                 }]
             },
             "TUESDAY",
-            4,
+            -1,
```

Comment:
What is this -1 signifying here? I know this is test data, but I am curious what it is symbolizing

Reply:
No match was found, see the comments above. Returns -1 if no match.